app/models/child_benefit_tax_calculator.rb

```diff
@@ -2,6 +2,7 @@
 
 class ChildBenefitTaxCalculator
   include ActiveModel::Validations
+  include Helper
```

Comment:
I've removed the Helper module. See updated commit note for b58e8557756b32748861372675f6d8156dffea56